README.md

```diff
@@ -107,6 +107,29 @@ need to unblock the pipeline by following the Buildkite run link in the PR and
 click on the corresponding
 [unblock pipeline button](https://buildkite.com/docs/pipelines/block-step).
 
+## Kani proofs
+We run Kani proofs for each PR and on merges to the main branch. The proofs
+check conformance of the virtio-queue implementation to requirements from the
+virtio specification. Different than a unit test, a proof is checked for every
+possible input thus enforcing conformance. To run all the proofs in all packages, use the
+following command:
```

Comment:
Looks good to me. I think adding [a link to kani's installation docs](https://model-checking.github.io/kani/install-guide.html#installing-the-latest-version) and saying it's required are worthwhile, otherwise people will get:

```sh
$ cargo kani
error: no such command: `kani`
```

Minor suggestion overall